css/main.css

```diff
@@ -1,3 +1,733 @@
+}
+
+.hero-image {
+  width: 100%;
+  height: 35rem;
+  background-image: url("../img/mobile-big-img.jpg");
+  background-position: center;
+  background-repeat: no-repeat;
+  background-size: cover;
+}
+
+.hero-content {
+  position: relative;
+  bottom: 33rem;
+  color: var(--white);
+}
+
+.logo-icon-words {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-evenly;
+  margin-top: 2rem;
+}
+
+.logo-icon img {
+  height: 10rem;
+  filter: invert(100%) sepia(4%) saturate(139%) hue-rotate(198deg)
+    brightness(117%) contrast(100%);
+}
+
+.logo-words {
+  border-bottom: 2px solid var(--white);
+  width: 4rem;
+  margin-bottom: 2.3rem;
+}
+
+.logo-words p {
+  font-size: 1rem;
+  font-weight: bolder;
+  letter-spacing: 0.5rem;
+  margin-bottom: 0.4rem;
+}
+
+.hero-content h1 {
+  font-size: 2.7rem;
+  margin-bottom: 1rem;
+  margin-top: 0.7rem;
+}
+
+.hero-content h2 {
+  font-size: 1.5rem;
+  font-weight: 200;
+  letter-spacing: 0.2rem;
+  padding: 0 2.5rem;
+  margin-bottom: 2.4rem;
+}
+
+.btn {
+  font-size: 1.1rem;
+  background-color: var(--red_brown);
+  color: var(--white);
+  border: none;
+  padding: 0.9rem 1.5rem;
+  cursor: pointer;
+}
+
+/* PRODUCTS */
+.feature {
+  margin: 30px 0;
+  text-align: center;
+  background-color: var(--white);
+}
+
+.feature-img img {
+  width: 100%;
+  height: 30rem;
+  object-position: center;
+  object-fit: cover;
+  position: relative;
+}
+
+.feature-title-btn {
+  position: absolute;
+  width: 100%;
+  padding: 1rem 0;
+  z-index: 1;
+}
+
+.feature-title-btn h2 {
+  margin-bottom: 0;
+  letter-spacing: 0.1rem;
+}
+
+.border-bottom-h2 {
+  width: 1rem;
+  background-color: var(--red_brown);
+  padding: 0.1rem 0.2rem;
+  margin: 1rem auto;
+}
+
+.about {
+  font-size: 1rem;
+  line-height: 1.7rem;
+}
+
+.about-us {
+  font-weight: 100;
+  font-size: 1.2rem;
+  letter-spacing: 0.7rem;
+  padding: 0;
+}
+
+.about-us-text {
+  padding: 4rem 1rem;
+}
+
+.img-about .img-area-about,
+.img-about .img-leaf-about {
+  width: 100%;
+  margin: -1rem 0;
+}
+
+.offer .offer-img {
+  width: 100%;
+  margin: -1.8rem 0;
+}
+
+.offer-online {
+  height: 25rem;
+  background-color: var(--green_black);
+  color: var(--white);
+}
+
+.offer-online .offer-online-h1 {
+  font-size: 3.2rem;
+  margin: -0.2rem 0;
+  padding-top: 4rem;
+}
+
+.offer-btn,
+.footer-subscribe-btn {
+  background-color: var(--white);
+  color: var(--green_black);
+}
+
+.offer-btn {
+  margin-top: 3rem;
+  padding: 0.9rem 3rem;
+}
+
+.form {
+  background-color: var(--off_white);
+  display: flex;
+  flex-direction: column;
+  padding: 0 1rem;
+}
+
+.form h3 {
+  font-size: 1.2rem;
+  font-weight: 100;
+  padding: 0 3rem;
+  letter-spacing: 0.4rem;
+}
+
+.form-section,
+.name {
+  display: flex;
+  flex-direction: column;
+  align-items: stretch;
+  text-align: left;
+  color: #000;
+}
+
+.form-section label {
+  padding: 1rem 0;
+  font-size: 1.3rem;
+}
+
+.form-section input {
+  padding: 1rem 0;
```

Comment:
It'd be good to add 1rem of padding on the left and right as well so the text doesn't touch the edges.